app/controllers/healthCheck.js

```diff
@@ -1 +0,0 @@
-exports.healthCheck = (_, res) => res.status(200).send({ uptime: process.uptime() });
```

Comment:
This is not requested, but it is an usefull method/endpoint that comes in the bootstrap. It is used to check the API health and is normally used in real projects. So, do not delete it :smile: